metagpt/actions/design_api.py

```diff
@@ -58,15 +58,15 @@
 """,
         "FORMAT_EXAMPLE": """
 [CONTENT]
-{
+{{
     "Implementation approach": "We will ...",
-    "project_name": "snake_game",
+    "Project name": "{project_name}",
```

Comment:
snake_game contribute this prompt as an example blog. we shouldn't populate context data

Reply:
During the design stage, the project name is already a valid string constant.